### PR TITLE
[APPS-2134] Remove moment.js dependency from start-task-cloud.compone…

### PR DESCRIPTION
**Pull request by rbahirsheth**
…nt.ts

**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [x] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
https://alfresco.atlassian.net/browse/APPS-2134

**What is the new behaviour?**
Removing the moment.js dependency from the component.

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/process-services-cloud/src/lib/task/start-task/components/start-task-cloud.component.ts [hunk 1]
```diff
@@ -179,11 +180,13 @@ export class StartTaskCloudComponent implements OnInit, OnDestroy {
     }
 
     onDateChanged(newDateValue) {
+        //console.log('from component', newDateValue);
```
Comment (MateuszHY) [inline lib/process-services-cloud/src/lib/task/start-task/components/start-task-cloud.component.ts:183]: Missed during cleanup or it's here for purpose ?

**Review by MateuszHY — CHANGES_REQUESTED** [review]

File: lib/process-services-cloud/src/lib/task/start-task/components/start-task-cloud.component.ts (hunk 1 above)
Comment (nikita-web-ua) [inline lib/process-services-cloud/src/lib/task/start-task/components/start-task-cloud.component.ts:183]: I guess can be deleted

**Review by MateuszHY — APPROVED** [review]

File: lib/process-services-cloud/src/lib/task/start-task/components/start-task-cloud.component.ts [hunk 2]
```diff
@@ -219,8 +221,8 @@ export class StartTaskCloudComponent implements OnInit, OnDestroy {
 
     public whitespaceValidator(control: UntypedFormControl) {
         const isWhitespace = (control.value || '').trim().length === 0;
-        const isValid = control.value.length === 0 || !isWhitespace;
-        return isValid ? null : { whitespace: true };
+        const isValidate = control.value.length === 0 || !isWhitespace;
```
Comment (MichalKinas) [inline lib/process-services-cloud/src/lib/task/start-task/components/start-task-cloud.component.ts:224]: Why did you just change the name of the variable here?

Comment (rbahirsheth) [reply lib/process-services-cloud/src/lib/task/start-task/components/start-task-cloud.component.ts:224]: I have imported & used isValid from date-fns which is giving an error here so, I have changed the variable name.

Comment (MichalKinas) [reply lib/process-services-cloud/src/lib/task/start-task/components/start-task-cloud.component.ts:224]: Okay in this case can you please name it like `isControlValid`? `isValidate` sounds a bit weird to me

**Review by AleksanderSklorz — APPROVED** [review]

File: lib/process-services-cloud/src/lib/task/start-task/components/start-task-cloud.component.ts (hunk 2 above)
Comment (rbahirsheth) [reply lib/process-services-cloud/src/lib/task/start-task/components/start-task-cloud.component.ts]: I have updated the variable name.

**Review by MichalKinas — APPROVED** [review]

**Review by eromano — APPROVED** [review]

**Review by DenysVuika — APPROVED** [review]

**Pull request merged**